allensdk/brain_observatory/behavior/session_apis/data_io/ophys_lims_api.py

```diff
@@ -516,5 +516,5 @@ def get_age(self) -> str:
 
 if __name__ == "__main__":
 
-    api = OphysLimsApi(789359614)
+    api = OphysLimsRawApi(789359614)
```

Comment:
that these files have `__main__`'s that seem to exist as examples is a little wonky. I think it would be more useful if the classes that a user is likely to instantiate had a section:
```
Examples
--------
```

Reply:
Yeah, I'm not sure why these have `__main__` sections. I can just get rid of them.